eigensystem.py

```diff
@@ -253,20 +253,27 @@ def single_file(input_filename):
               " minI="+"{:.2e}".format(np.min(np.imag(eigenvalues_thisr))),
               " maxI="+"{:.2e}".format(np.max(np.imag(eigenvalues_thisr))))
     loopend=time.time()
-    print("total time ="+loopstart-loopend+" seconds")        
+    print("total time ="+"{:.2e}".format(loopend-loopstart)+" seconds")        
+
+        
 
     #----------
     # get mu_tilde for output
     #----------
     S_nok, mu_tilde = get_shared_numpy_arrays()
 
+    #----------
```

Comment:
I think this is data reduction, and would be better in plotting scripts than in the main script. Simpler = less to maintain.